### PR TITLE
feat: knockout gene tooltip

### PR DESCRIPTION
**Pull request by AlbaLopez**

### CODE REVIEW

File: src/Behavior.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ function toggle_reaction_hover(on_off) {
   if (on_off) {
     this.reaction_mouseover = (d) => {
       if (!this.dragging) {
-        this.map.callback_manager.run('show_tooltip', null, 'reaction_label', d);
+        //this.map.callback_manager.run('show_tooltip', null, 'reaction_label', d);
```
Comment (kvikshaug) [inline src/Behavior.js]: Can you please explain why this change is necessary? And would it not be better to remove the whole function, since it now doesn't do anything?

Comment (AlbaLopez) [reply src/Behavior.js]: @annasirunian  could you please delete this line of code?

Comment (annasirunian) [reply src/Behavior.js]: Done
